### PR TITLE
Add initial package structure

### LINKED ISSUE #3: Make an initial python module
Make an (almost) empty python module that can be installed using `pip install .` in the repository and write at least one test that can be ran by using `pytest`.

### PR DESCRIPTION
**Pull request by haavardnk**
Created initial scaffolding for package.

Questions for setup.py:
- What should author be? Currently "Software Innovation Bergen, Equinor ASA"
- Is there any better way of doing the versioning?

Closes #3

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+def _load_readme():
+    readme = os.path.join(PROJECT_ROOT, "README.md")
+    with open(readme) as f:
+        return f.read()
+
+
+setup(
+    name="configsuite-tui",
+    author="Software Innovation Bergen, Equinor ASA",
```
Comment (markusdregi) [inline setup.py:15]: Well, since your not part of Software Innovation Bergen, I suggest we leave this as Equinor ASA 🤷

**Comment by markusdregi** [comment]
When it comes to versioning, we typically use `scm_versioning` based on the GitHub tags as done here: https://github.com/equinor/configsuite/blob/b2b3d7f5b497a127af2fe2620f4d8e40c9734380/setup.py#L45

File: configsuite_tui/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .tui import *
```
Comment (markusdregi) [inline configsuite_tui/__init__.py:1]: Make git, GitHub and the universe happy by adding newlines where missing

**Comment by markusdregi** [comment]
**Challenge of the day:**

The commit in this PR does many things at the same time, although the combined footprint is very small. As an exercise I encourage you to split the commit into at least two parts; one editing the `.gitignore` and another doing the rest. `git rebase` is your friend here...

Also, the comment in the previous PR on git messages apply

**Comment by haavardnk** [comment]
Changes made and commits split, check if OK

**Review by markusdregi — APPROVED** [review]
Well done! 👏 Happy to see that you managed the `git`-rebasing 👍

**Pull request merged**